tests/continuous/test_standard.py

```diff
@@ -25,6 +25,7 @@ def test_mse_xarray_1d():
     result = scores.continuous.mse(fcst_as_xarray_1d, obs_as_xarray_1d)
 
     expected = xr.DataArray(1.0909)
+    assert isinstance(result, xr.DataArray)
     assert result.round(PRECISION) == expected.round(PRECISION)
```

Comment:
Why not just use xarray.testing.assert_equal or xarray.testing.assert_allclose?

Reply:
There are several cases below where you could take the xarray.testing... approach

Reply:
Is there a specific reason to prefer one way or the other? I think I just tend not to pick up something specific when a more simple/general way will do. I don't have any strong opinion here, I just don't know really if there are clear reasons to do one thing or another.

Reply:
I appreciate the feedback. In order to more rapidly reduce the number of linting/style errors, I'm going to go ahead and merge this as it's functionally ready, and I'll organise a time to discuss this question with the dev team and get a position on whether there's a clear best practice approach.

Reply:
Fair enough since it works functionally.

Advantages in this situation is that it is cleaner since it tests both those assert statements in a single line.

Later on in that file, outside the changes in this pull request, there are additional statements like `assert result.dims == expected_dimensions` https://github.com/nci/scores/blob/develop/tests/continuous/test_standard.py#L107. If `xr.testing` is used, then it test all of this at once and additionally tests if things like if the coordinates match which isn't tested in the current implementation.